src/components/WriteEditModal/ImageInput.tsx

```diff
@@ -1,53 +1,47 @@
 'use client';
 
-import React, { useState } from 'react';
+import { useState, ChangeEvent } from 'react';
 import Image from 'next/image';
-import Carmera from '@/public/svgs/camera.svg';
-import DeleteImage from '@/public/svgs/deleteImage.svg';
+import CarmeraIcon from '@/public/svgs/camera.svg';
+import DeleteImageIcon from '@/public/svgs/deleteImage.svg';
 import classNames from 'classnames/bind';
 import styles from './ImageInput.module.scss';
 
-export default function ImageInput() {
-  const cn = classNames.bind(styles);
+const cn = classNames.bind(styles);
 
+export default function ImageInput() {
   const [selectedImageFile, setSelectedImageFile] = useState<string[]>([]);
-  const handleChangeImage = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (selectedImageFile.length === 4) {
+  const handleChangeImage = (e: ChangeEvent<HTMLInputElement>) => {
+    const { files } = e.target;
+    if (selectedImageFile.length >= 4 || !files) {
       alert('이미지 4개 이상 불가');
       return;
     }
-    if (e.target.files) {
-      const file = e.target.files[0];
-      const imageUrl = URL.createObjectURL(file);
-      setSelectedImageFile([...selectedImageFile, imageUrl]);
-    }
+    const imageUrl = URL.createObjectURL(files[0]);
+    setSelectedImageFile([...selectedImageFile, imageUrl]);
```

Comment:
여기도 prev로 접근하는 게 좋을 것 같아요!
```suggestion
    setSelectedImageFile((prev) => [...prev, imageUrl]);
```